### ISSUE TITLE
2 notifications displayed if clicking the Invite button, after a contribution was made 

### ISSUE THREAD

**Issue by viorelaioia** [mozilla/voice-web#2323]
**STR:**

1. Sign up to the challenge
2. Navigate to Listen page and validate a sentence
3. Navigate to [Challenge](https://voice.allizom.org/en/dashboard/challenge?date=2019-11-20) page and click Invite button
4. Click "X" button

**Expected:** 
"50+ points for first invite" notification is displayed

**Actual:**
2 notifications are displayed on the page. These notifications are displayed every time I click the Invite button, not only the first time.

![image](https://user-images.githubusercontent.com/5293100/68859429-e65c9900-06ef-11ea-9b09-b2b7138705a2.png)

**Comment by rileyjshaw** [comment]
Ah, good catch! @mbransn @georai is it okay to only show one notification here? I think adding timing delays, etc. is out of scope.

**Comment by mbransn** [comment]
Yep, I'd go with whichever was initiated first in the order for primary display.

**Comment by catherine917** [comment]
I think it's similar to the issue https://github.com/mozilla/voice-web/issues/2319.

**Comment by viorelaioia** [comment]
This issue is still happening, I managed to reproduce it in prod.

**Comment by catherine917** [comment]
@viorelaioia 

> Sign up to the challenge
Navigate to Listen page and validate a sentence
Navigate to Challenge page and click Invite button
Click "X" button

As you said above, you should get two achievements. Therefore, two notifications display on the dashboard is correct. 

> These notifications are displayed every time I click the Invite button, not only the first time.

This is an issue and I think it should be fixed in the PR https://github.com/mozilla/voice-web/pull/2334. 

> This issue is still happening, I managed to reproduce it in prod.

I don't know which one is still happening. I tested in my local server and clicked the `invite` button several times, the two notifications showed up only once.

**Comment by mbransn** [comment]
@viorelaioia @rleitan can you retest tomorrow?

**Comment by viorelaioia** [comment]
@mbransn , if getting 2 notifications is the expected result after following the steps from the issue, then I can confirm that is working as expected. Also, the 2 notification are showing only once.

**Comment by mbransn** [comment]
Got it, after experiencing the two notifications on my end in a positive way, I can confirm that this is fixed. @phirework please close when you are comfortable!
